lib/core/_layout.scss

```diff
@@ -41,20 +50,27 @@
     //
     // Generates box row to float boxes inside
     //
-    [#{$hill-layout-attribute-name}*="row"]  {
-        & > [#{$hill-layout-attribute-name}*="box-"] {
+    [#{$hill-html-prefix}-layout*="row"]  {
+        & > [#{$hill-html-prefix}-layout*="box-"] {
             float: left;
         }
 
-        @extend %_clearfix;
+        &:before,
```

Comment:
`layout="clear"` and `layout="row"` using the same clearfix code. Why not creating a public mixin in `lib/mixins` called `hill-clearfix()` ?